tests/scripts/mbedtls_test.py

```diff
@@ -90,20 +89,24 @@ def __parse(self, data_f):
         :param data_f: Data file object
         :return:
         """
-        for line in data_f:
-            line = line.strip()
+        while True:
```

Comment:
Did this loop actually need to change? Doesn't `for line in file` still work in python3?

Reply:
Once I had to change to `readline()` I got the error described in https://stackoverflow.com/questions/826493/mixing-files-and-loops